pkg/proxy/nftables/proxier.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package nftables
 
 //
-// NOTE: this needs to be tested in e2e since it uses iptables for everything.
+// NOTE: this needs to be tested in e2e since it uses nftables for everything.
```

Comment:
Looking forward - how can we get better and more robust e2e tests of specific k-p functionality.  It feels like that should be seperable from main e2e, no?